core/sched.c

```diff
@@ -118,7 +118,14 @@ int __attribute__((used)) sched_run(void)
 #ifdef MODULE_SCHEDSTATISTICS
         schedstat *active_stat = &sched_pidlist[active_thread->pid];
         if (active_stat->laststart) {
-            active_stat->runtime_ticks += now - active_stat->laststart;
+            /* ensure now > laststart, to avoid flawed sum of runtime ticks
```

Comment:
IMO we can remove this check. It doesn't catch all overflows anyways. Worse, it drops indeterministic periods depending on when the thread got scheduled.
I think that losing 71minute overflows, and documenting that properly, is preferable than this.